src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const operations = {
+    sum: '+', subtract: '-', multiplication: 'x', division: '÷', percentage: '%',
+  };
+  const x = Big(numberOne);
+  const y = Big(numberTwo);
+  let result = null;
+  switch (operation) {
+    case operations.sum:
+      result = x.plus(y);
+      break;
+    case operations.subtract:
+      result = x.minus(y);
+      break;
+    case operations.multiplication:
+      result = x.times(y);
+      break;
+    case operations.division:
+      result = x.div(y);
+      break;
```

Comment:
when we divide by ZERO, we get an error...manage that ;), you can display `INFINITY` on screen
![Screen Shot 2020-04-05 at 3 54 03 PM](https://user-images.githubusercontent.com/40712459/78501784-3991bf80-7755-11ea-857a-fec423ffea58.png)